lib/Locking/Provisioning.php

```diff
@@ -89,78 +89,78 @@ protected function getPath($parameters) {
 	}
 
 	/**
-	 * @return \OC_OCS_Result
+	 * @return \OC\OCS\Result
 	 */
 	public function isLockingEnabled() {
 		try {
 			$this->getLockingProvider();
-			return new \OC_OCS_Result(null, 100);
+			return new \OC\OCS\Result(null, 100);
 		} catch (\RuntimeException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_NOT_IMPLEMENTED, $e->getMessage());
+			return new \OC\OCS\Result(null, Http::STATUS_NOT_IMPLEMENTED, $e->getMessage());
 		}
 	}
 
 	/**
 	 * @param array $parameters
-	 * @return \OC_OCS_Result
+	 * @return \OC\OCS\Result
 	 */
 	public function acquireLock(array $parameters) {
 		try {
 			$path = $this->getPath($parameters);
 		} catch (NoUserException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_NOT_FOUND, 'User not found');
+			return new \OC\OCS\Result(null, Http::STATUS_NOT_FOUND, 'User not found');
 		} catch (NotFoundException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_NOT_FOUND, 'Path not found');
+			return new \OC\OCS\Result(null, Http::STATUS_NOT_FOUND, 'Path not found');
 		}
 		$type = $this->getType($parameters);
 
 		$lockingProvider = $this->getLockingProvider();
 
 		try {
 			$lockingProvider->acquireLock($path, $type);
-			$this->config->setAppValue('testing', 'locking_' . $path, $type);
-			return new \OC_OCS_Result(null, 100);
+			$this->config->setAppValue('testing', 'locking_' . $path, (string) $type);
+			return new \OC\OCS\Result(null, 100);
 		} catch (LockedException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_LOCKED);
+			return new \OC\OCS\Result(null, Http::STATUS_LOCKED);
 		}
 	}
 
 	/**
 	 * @param array $parameters
-	 * @return \OC_OCS_Result
+	 * @return \OC\OCS\Result
 	 */
 	public function changeLock(array $parameters) {
 		try {
 			$path = $this->getPath($parameters);
 		} catch (NoUserException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_NOT_FOUND, 'User not found');
+			return new \OC\OCS\Result(null, Http::STATUS_NOT_FOUND, 'User not found');
 		} catch (NotFoundException $e) {
-			return new \OC_OCS_Result(null, Http::STATUS_NOT_FOUND, 'Path not found');
+			return new \OC\OCS\Result(null, Http::STATUS_NOT_FOUND, 'Path not found');
 		}
 		$type = $this->getType($parameters);
 
 		$lockingProvider = $this->getLockingProvider();
 
 		try {
 			$lockingProvider->changeLock($path, $type);
-			$this->config->setAppValue('testing', 'locking_' . $path, $type);
-			return new \OC_OCS_Result(null, 100);
+			$this->config->setAppValue('testing', 'locking_' . $path, (string) $type);
```

Comment:
``$type`` is ``int`` but ``setAppValue()`` expects ``string``. ``phan`` reported this. We can explicitly cast to ``string`` to make it clear that we know we are doing this.